supervisor/process.py

```diff
@@ -209,6 +211,9 @@ def spawn(self):
         self._assertInState(ProcessStates.EXITED, ProcessStates.FATAL,
                             ProcessStates.BACKOFF, ProcessStates.STOPPED)
 
+        if delaystart:
+            time.sleep(self.config.startdelay)
```

Comment:
The startdelay gets compounded for every process, (proc n gets a n*startdelay seconds delay) and it only happens the first time it starts (I don't believe it will mess with the async operation after startup)